sdk/src/Services/S3/Custom/Transfer/_async/TransferUtility.async.cs

```diff
@@ -54,6 +54,8 @@ namespace Amazon.S3.Transfer
     /// </remarks>
     public partial class TransferUtility : ITransferUtility
     {
+        private readonly string S3TransferTracerScope = "AWSSDK.S3.Transfer";
```

Comment:
Can we reference this prefix in this string instead of making another copy?
https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Core/Amazon.Runtime/Telemetry/TelemetryConstants.cs#L26

Reply:
Done